### DISCUSSION TITLE
Lay-Z-Spa Helsinki (2021) detailed photographs

### DISCUSSION THREAD

**Discussion by owenashurst** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#567] [General]
So with my recent success with this project, I've taken some detailed photographs of the connections within the Helsinki model pump (none round version).

Connectors:
----------------------------------------------------------

![PXL_20230623_125526038.jpg](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/4403436/74026db8-3fb8-4ae6-a131-9c315aa27ab1)

![PXL_20230623_125800249.jpg](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/4403436/5394d4b3-59a7-45dd-a025-9dcd5a92f138)

![PXL_20230623_125511168.jpg](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/4403436/ae6efd68-8fba-47c4-8fec-d7b3b8f4fc81)

![PXL_20230623_125536989.jpg](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/4403436/b656cf10-9d7b-4d29-af96-a451f9656f15)

![PXL_20230623_125753128.jpg](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/4403436/e38b54dc-e314-4dd0-a56d-0d569336126c)

Lay-Z-Spa module connected:
----------------------------------------------------------

![PXL_20230705_170000853.jpg](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/4403436/903cb0e9-2c42-4d2a-8baf-23c3c68b608c)

![PXL_20230705_170004030.jpg](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/4403436/804c317c-2051-4528-b35d-1a28ac60f81a)

**Comment by visualapproach** [comment]
Nice

**Comment by kallejoken** [comment]
> So with my recent success with this project, I've taken some detailed photographs of the connections within the Helsinki model pump (none round version).
> 
> ## Connectors:
> ![PXL_20230623_125526038.jpg](https://user-images.githubusercontent.com/4403436/252440485-74026db8-3fb8-4ae6-a131-9c315aa27ab1.jpg)
> 
> ![PXL_20230623_125800249.jpg](https://user-images.githubusercontent.com/4403436/252440483-5394d4b3-59a7-45dd-a025-9dcd5a92f138.jpg)
> 
> ![PXL_20230623_125511168.jpg](https://user-images.githubusercontent.com/4403436/252440484-ae6efd68-8fba-47c4-8fec-d7b3b8f4fc81.jpg)
> 
> ![PXL_20230623_125536989.jpg](https://user-images.githubusercontent.com/4403436/252440486-b656cf10-9d7b-4d29-af96-a451f9656f15.jpg)
> 
> ![PXL_20230623_125753128.jpg](https://user-images.githubusercontent.com/4403436/252440482-e38b54dc-e314-4dd0-a56d-0d569336126c.jpg)
> 
> ## Lay-Z-Spa module connected:
> ![PXL_20230705_170000853.jpg](https://user-images.githubusercontent.com/4403436/252440036-903cb0e9-2c42-4d2a-8baf-23c3c68b608c.jpg)
> 
> ![PXL_20230705_170004030.jpg](https://user-images.githubusercontent.com/4403436/252440114-804c317c-2051-4528-b35d-1a28ac60f81a.jpg)

What ist the Modellnumber of the pump?

**Reply by owenashurst** [reply]
I'm not sure, why? I've installed this on two of the same type of pumps and it works fine if you're concerned.
